devices/stm32h7a3.yaml

```diff
@@ -233,9 +236,19 @@ MDMA:
   _include:
     - collect/dma/mdma.yaml
 
-OCTOSPI2:
+OCTOSPI1:
+  WCCR:
+    _modify:
+      IMODE:
+        bitWidth: 3
+      IDTR:
+        bitOffset: 3
+      ISIZE:
+        bitOffset: 4
+        bitWidth: 2
```

Comment:
At what point should things like this get its own file? I see it is the same for all rm0455 devices